pkg/controller/syncsetinstance/syncsetinstance_controller.go

```diff
@@ -303,6 +312,19 @@ func (r *ReconcileSyncSetInstance) Reconcile(request reconcile.Request) (reconci
 	return reconcile.Result{RequeueAfter: reapplyDuration}, nil
 }
 
+func (r *ReconcileSyncSetInstance) hasSourceSecretsInAnotherNamespace(ssi *hivev1.SyncSetInstance, spec *hivev1.SyncSetCommonSpec) bool {
+	// This check is not relevant for SelectorSyncSets
+	if ssi.Spec.SyncSetRef == nil {
```

Comment:
Do we need this check? This was already checked by the caller. Can we either remove it here or remove it at the caller location?